### PR TITLE
use american english and fix typos

### PR DESCRIPTION
**Pull request by QWERTZ11**

### CODE REVIEW

**Comment by QWERTZ11** [comment]
Hi @christophM , the additional information for the difference is as follows:

03-00-feature-importance.Rmd
6-th row: black**-**box -->  black box 



03-6-fi-correlated.Rmd 
7-th row: The last second sentence:
In the case of correlated features in the data,...  --> I**n the** case of correlated features in the data,...

31-th row: The second sentence:
...where additional dependencies between each **features**... --> ...where additional dependencies between each **feature**... 

60-th row: 
As **a** performance measures we decided showing two.  --> As performance measures we decided showing two. 

111-th row: colour --> color

151-th row: 
For example, the point is still in the blue shaded **are**. --> For example, the point is still in the blue shaded **area**.

333-th row: 
Once again, we can observe a drop in importance of $X1$ and $X2$ --> Once again, we can observe a drop in **the** importance of $X1$ and $X2$

**Comment by QWERTZ11** [comment]
02-00-ale.Rmd 34-th row: section 3.x --> section \@ref(ale-misc)

File: 04-09-lime-neighbourhood.Rmd [hunk 1]
```diff
@@ -449,7 +449,7 @@ It seems as if the predictive surface is too complex to facilitate local and sta
 The curse of dimensionality affects locality very strongly.
 As distances in higher-dimensional Euclidean space are increasingly less meaningful, the definition of locality is very poor with an increasing number of features.
 
-Summarising, we observe both effects being described in the literature also for our real data example: 
+summarizing, we observe both effects being described in the literature also for our real data example:
```
Comment (SebGruber1996) [inline 04-09-lime-neighbourhood.Rmd]: Capitalize

File: 04-00-lime.Rmd [hunk 2]
```diff
@@ -91,26 +91,26 @@ $x$ can be an instance that is entirely new to us as long as it can be represent
...
 $f$ refers to the black box model we want to explain and $g$ to the explainer. 
 $G$ represents the complete hypothesis space of a given interpretable learner.
-The explanation has to deal with two trade-off terms when minimising: 
+The explanation has to deal with two trade-off terms when minimizing: 
 The first term $\mathcal{L}\left(f, g, \pi_x \right)$ is responsible to deliver the optimal fit of $g$ to the model $f$ while a low _loss_ is desirable indicating high (local) _fidelity_.
-The optimal fit is only found with respect to a proximity measure $\pi_x(z)$ in the neighbourhood of $x$.
+The optimal fit is only found with respect to a proximity measure $\pi_x(z)$ in the neighborhood of $x$.
 
-### Neighbourhood
+### neighborhood
```
Comment (SebGruber1996) [inline 04-00-lime.Rmd]: capitalize

**Comment by christophM** [comment]
Thanks @QWERTZ11 and @SebGruber1996

**Comment by QWERTZ11** [comment]
You are welcome！

**Pull request merged**